python-dali (0.6) unstable; urgency=low

  * Python 3 fix in async Tridonic driver
  * Updated Hasseb driver

 -- Stephen Early <steve@assorted.org.uk>  Sun, 11 Nov 2018 17:48:00 +0000

python-dali (0.5) unstable; urgency=low

  * Move tests to dedicated package.
  * Rename ``dali.device.Device._addressobj`` to
    ``dali.device.Device.address_obj`` since it is used as API in
    ``dali.command.Command``.
  * Move ``dali.device.Device`` class to ``dali.bus``, ``dali.device`` module
    has been removed which conflicted with ``dali.device`` package.
  * Add ``dali.exceptions`` module and move all DALI related exceptions to it.
  * Move ``dali.interfaces`` to ``dali.drivers.daliserver``.
  * Update Package structure documentation.

 -- Robert Niederreiter <rnix@squarewave.at>  Tue, 02 May 2017 12:50:08 +0200

python-dali (0.4) unstable; urgency=low

  * PyPI support; rename package from dali to python-dali to avoid clash

 -- Stephen Early <steve@assorted.org.uk>  Sun, 19 Mar 2017 19:33:00 +0000

python-dali (0.3) unstable; urgency=low

  * Python 3 support.

 -- Stephen Early <steve@assorted.org.uk>  Tue, 15 Nov 2016 14:35:00 +0000

python-dali (0.2) unstable; urgency=low

  * Introduce basic sync and async drivers.
  * Basic implementation of LUNATONE Dali USB driver.

 -- Robert Niederreiter <rnix@squarewave.at>  Mon, 20 Jun 2016 10:57:00 +0100

python-dali (0.1) unstable; urgency=low

  * Initial release.

 -- Stephen Early <steve@greenend.org.uk>  Mon, 27 Apr 2015 18:05:00 +0100
